dali/dfuXRefLib/XRefFilesNode.cpp

```diff
@@ -431,7 +436,7 @@ bool CXRefFilesNode::AttachPhysical(const char *Partmask,IUserDescriptor* udesc,
             partProps->setPropInt64("@size", physicalSize);
         totalSize += physicalSize;
 
-        fileDesc->setPart(partNo, node,_filename.str(), partProps);
+        fileDesc->setPart(partNo, node, filename, partProps);
```

Comment:
not new/not for now, but this is setting overridename for every part.
Assuming the file it's attaching all have the same 'mask' (surely they should?),
then it shouldn't need to call setPart N times like this, but instead call setPartMask with the mask